### PR TITLE
Documented the workflow of the contract states

### PR DESCRIPTION
**Pull request by amatmv**
### New section added: states of a contract

Added documentation of the contract states, with workflows diagrams and the states histogram.

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by gdalmau** [comment]
@amatmv For future documentations, use screenshots in Spanish

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

File: docs/distri/polizas.md [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+Quan les dades estàn validades i són correctes, l'estat de la pòlissa passa a
+ser **Activa**. Com indica el nom, l'estat de Activa indica que una pòlissa
+està en vigor; que s'hi poden fer modificacions contractuals, i passar a altres 
+estats com Impagament.
+
+Des de **Impagament** es podrà o bé donar de baixa la pòlissa, passar en estat
+de **Tall**, o bé reactivar-la.
+
+Des de Activa, també es poden fer **Modificacions Contractuals**. Per una
+explicació més detallada d'aquestes, consultar [aquesta](facturacion.md) pàgina.
```
Comment (jaumef) [inline docs/distri/polizas.md]: Millor posar el link específic. I.e.:

```markdown
consultar la página de [modificaciones contractuales](facturacion.md#modificaciones-contractuales).
```

File: docs/distri/polizas.md [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
+
+També es podrà donar de **Baixa** una pòlissa o **Cancel·lar-la**.
```
Comment (jaumef) [inline docs/distri/polizas.md]: Aquesta línia no es veu. Hauries de posar un salt de línia entre text i notes.

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by ecarreras** [comment]
This documentation is not only exclusive of **distribuidora** I think this should go in **Base**

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_polissa_workflow/

**Pull request merged**